services/web/server/src/simcore_service_webserver/projects/projects_nodes_handlers.py

```diff
@@ -99,6 +101,71 @@ async def get_node(request: web.Request) -> web.Response:
         raise web.HTTPNotFound(reason=f"Project {project_uuid} not found") from exc
 
 
+@routes.get(f"/{VTAG}/projects/{{project_id}}/nodes/{{node_id}}/resources")
+@login_required
+@permission_required("project.node.read")
+async def get_node_resources(request: web.Request) -> web.Response:
+    user_id: UserID = request[RQT_USERID_KEY]
+
+    try:
+        project_uuid = ProjectID(request.match_info["project_id"])
+        node_uuid = NodeID(request.match_info["node_id"])
+    except KeyError as err:
+        raise web.HTTPBadRequest(reason=f"Invalid request parameter {err}") from err
+    try:
+        # ensure the project exists
+        project = await projects_api.get_project_for_user(
+            request.app,
+            project_uuid=f"{project_uuid}",
+            user_id=user_id,
+            include_templates=True,
+        )
+
+        resources = await projects_api.get_project_node_resources(
+            request.app, project=project, node_id=node_uuid
+        )
+        return web.json_response({"data": resources}, dumps=json_dumps)
+
+    except ProjectNotFoundError as exc:
+        raise web.HTTPNotFound(reason=f"Project {project_uuid} not found") from exc
```

Comment:
THOUGHT: shouldn't we have a kind of "friedly-message" in the projects' errors that we can reuse in reason? something like

```python
raise web.HTTPNotFound(reason=exc.user_message) from exc
```